weblab/static/js/compare.js

```diff
@@ -595,29 +595,17 @@ function parseUrl (event)
 {
 	var entityIds = null;
   var parts = document.location.pathname.split("/")
-	//var t = document.location.href.split ("/");
-	//console.log (t);
 
 	for (var i = 0; i < parts.length; i++)
 	{
-    /*
-		if ("/" + t[i] == contextPath && t[i+2] == "m")
-		{
-			basicurl = t.slice (0, i + 3).join ("/") + "/";
-			entityType = "model";
-			entityIds = t.slice (i + 3);
-		}
-		if ("/" + t[i] == contextPath && t[i+2] == "p")
-		{
-			basicurl = t.slice (0, i + 3).join ("/") + "/";
-			entityType = "protocol";
-			entityIds = t.slice (i + 3);
-		}
-    */
     if (parts[i] == 'experiments') {
       basicurl = parts.slice(0, i+2).join('/') + '/';
       entityType = 'experiment';
       entityIds = parts.slice(i+2);
+    } else if (parts[i] == 'models' && parts[i+1] == 'compare') {
```

Comment:
What about comparing two versions of a protocol? BiVeS doesn't make much sense in this case, but unix diff would. (Note that I don't think it's worth restricting what we offer depending on entity type: offer both, and users can figure out which is helpful for them!)

Reply:
Yes, the URLS etc. are set up to support protocols, so this bit of code can easily be adapted to do so.